test/integration/rules/aria-valid-attr-value/aria-valid-attr-value.html

```diff
@@ -51,6 +51,46 @@ <h2>Violations</h2>
   <div aria-posinset="-22" id="violation42">hi</div>
 
   <div aria-setsize="-22" id="violation43">hi</div>
+
```

Comment:
Can you add an example with a hidden error, that passes because it has aria-invalid="false"?